src/main/webapp/resources/angular-templates/admin/partials/event/detail.html

```diff
@@ -64,11 +82,11 @@ <h5 class="text-muted">Here the categories that have been defined for this event
                                 </div>
                                 <div class="hidden-xs hidden-sm col-md-4">
                                     <uib-progress max="getActualCapacity(ticketCategory, event)" uib-tooltip-template="'categoryStats.html'" tooltip-popup-close-delay="500">
-                                        <uib-bar value="ticketCategory.checkedInTickets" type="success"><span>{{ticketCategory.checkedInTickets}}</span></uib-bar>
-                                        <uib-bar value="ticketCategory.soldTickets" type="warning"><span>{{ticketCategory.soldTickets}}</span></uib-bar>
-                                        <uib-bar value="ticketCategory.pendingTickets" type="info" class="slice-pending">{{ticketCategory.pendingTickets}}</uib-bar>
-                                        <uib-bar value="ticketCategory.notSoldTickets" type="info">{{ticketCategory.notSoldTickets}}</uib-bar>
-                                        <uib-bar ng-if="!ticketCategory.bounded" value="event.dynamicAllocation" type="default">{{event.dynamicAllocation}}</uib-bar>
+                                        <uib-bar id="checkedIn" value="ticketCategory.checkedInTickets" type="success"><span>{{ticketCategory.checkedInTickets}}</span></uib-bar>
```

Comment:
don't use id="", as they are in a ng-repeat, there will be then multiple element with the same id, which is obviously wrong. Switch to class.